tools/donate-cpu.py

```diff
@@ -168,19 +168,16 @@ def wget(url, destfile, bandwidth_limit):
         print('wget failed with ' + str(exitcode))
         os.remove(destfile)
         return False
-    if os.path.isfile(destfile):
-        return True
-    print('Sleep for 10 seconds..')
-    time.sleep(10)
-    return False
+    if not os.path.isfile(destfile):
+        return False
+    return True
 
 
 def downloadPackage(workPath, package, bandwidth_limit):
     print('Download package ' + package)
     destfile = workPath + '/temp.tgz'
     if not wget(package, destfile, bandwidth_limit):
-        if not wget(package, destfile, bandwidth_limit):
```

Comment:
Why did you remove the retry?

Reply:
I suggest adding a loop with 3 retries and a short pause between them here. Then it is more obvious i guess.

Reply:
As wget is already called with `--tries` which will handle recoverable errors. In case the file doesn't even exist on the FTP (which I encountered a few times) it will just try to download it again and fail. I asked danmar if there was a reason for do this twice and apparently there is none.

Reply:
Ok